kubernetes/zenko/charts/mgob/values.yaml

```diff
@@ -0,0 +1,83 @@
+# Default values for zenko.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+mgob:
+  enabled: true
+  replicaCount: 1
+  image:
+    repository: stefanprodan/mgob
+    pullPolicy: IfNotPresent
+    tag: edge
+  persistentVolume:
+    enabled: true
+    accessModes:
+      - ReadWriteOnce
+    annotations: {}
+    existingClaim: ""
+    size: 10Gi
+
+  configMap:
+    name: "mgob-config"
```

Comment:
The ConfigMap name should also be using the templated name. It's okay to have an optional value to point to a custom ConfigMap but this shouldn't be the default. (See other comments about static names)